cli/bzk/service.go

```diff
@@ -0,0 +1,422 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/jawher/mow.cli"
+
+	docker "github.com/bywan/go-dockercommand"
+
+	dockerclient "github.com/fsouza/go-dockerclient"
+)
+
+var allContainers []dockerclient.APIContainers
+
+func startService(cmd *cli.Cmd) {
+	cmd.Spec = "[--home|--scm-key|--registry|--docker-sock]... [--tag]"
+
+	bzkHome := cmd.String(cli.StringOpt{
+		Name:   "home",
+		Desc:   "Bazooka's work directory",
+		EnvVar: "BZK_HOME",
+	})
+	scmKey := cmd.String(cli.StringOpt{
+		Name:   "scm-key",
+		Desc:   "Location of the private SSH Key Bazooka will use for SCM Fetch",
+		EnvVar: "BZK_SCM_KEYFILE",
+	})
+	registry := cmd.String(cli.StringOpt{
+		Name:   "registry",
+		EnvVar: "BZK_REGISTRY",
+	})
+	dockerSock := cmd.String(cli.StringOpt{
+		Name:   "docker-sock",
+		Desc:   "Location of the Docker unix socket, usually /var/run/docker.sock",
+		EnvVar: "BZK_DOCKERSOCK",
+	})
+	tag := cmd.String(cli.StringOpt{
+		Name: "tag",
+		Desc: "Tag on which run bazooka",
```

Comment:
A more readable description ? Something like "The bazooka version to run" maybe ?